app/src/main/java/com/kamron/pogoiv/updater/DownloadUpdateService.java

```diff
@@ -14,13 +14,13 @@
 import android.support.v4.content.FileProvider;
 
 import com.kamron.pogoiv.BuildConfig;
+import com.kamron.pogoiv.R;
 
 import java.io.File;
 
 public class DownloadUpdateService extends Service {
 
     public static final String FILE_NAME = "GoIV_new.apk";
-    public static final String DOWNLOAD_UPDATE_TITLE = "Updating GoIV";
```

Comment:
Since you removed this constant, you should update also MainActivity.java:141 to:
`if (fileName.equals(getString(R.string.notification_updating))) {`